### PR TITLE
Changes couchdb-cluster to be an entity

### PR DESCRIPTION
**Pull request by m4rkmckenna**
* Created couchdb-cluster-template

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[*]
```
Comment (Graeme-Miller) [inline .editorconfig]: not sure about this file, should we add a git ignore for this

**Review by Graeme-Miller — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (m4rkmckenna) [reply .editorconfig]: Its a standard thing http://editorconfig.org/ so no

Comment (Graeme-Miller) [reply .editorconfig]: but is it something that is standard in brooklyncentral?
If not should it be and does it warrant a discussion on the mailing list

Comment (m4rkmckenna) [reply .editorconfig]: There is no mailing list for brooklyncentral

Comment (Graeme-Miller) [reply .editorconfig]: of course, but there is one for Brooklyn. Whilst this isn't a part of the open source project they are linked and used by the community. I wonder if it is worth discussing with the Brooklyn mailing list about using editorconfig or similar as a standard

**Pull request merged**